Swat/SwatTextareaEditor.php

```diff
@@ -141,11 +141,17 @@ public function __construct($id = null)
         $this->requires_id = true;
         $this->rows = 30;
 
-        if (self::$tiny_mce_api_key !== null && !empty(self::$tiny_mce_api_key)) {
-            $tiny_mce_url = "https://cdn.tiny.cloud/1/".
-                self::$tiny_mce_api_key."/tinymce/5/tinymce.min.js";
+        if (
+            self::$tiny_mce_api_key !== null &&
```

Comment:
`empty(null)` returns true, so technically you don't need this. Typically though, we avoid using `empty()` and just use `=== null`. You're doing this in case there's an empty string, right? Can't we avoid that ever by just using the default config value of null?